os-build/debian/control

```diff
@@ -0,0 +1,15 @@
+Source: sozu
+Section: net
+Priority: optional
+Maintainer: Alessio Coltellacci <alessio.coltellacci@clever-cloud.com>
+Build-Depends: debhelper (>= 10),
+               dh-cargo,
+               dh-systemd,
+Standards-Version: 4.1.2
+Homepage: https://www.sozu.io/
+
+Package: sozu
+Architecture: amd64
+Depends: ${shlibs:Depends}, ${misc:Depends}
```

Comment:
there should be at least a dependency on openssl, probably other libs too

Reply:
I know it's in WIP